docs/download-apis.js

```diff
@@ -0,0 +1,28 @@
+const download = require("download");
+const versions = require("./data/versions.json");
+
+const renderMultiVersion = process.argv[2];
+
+docVersions = renderMultiVersion
+	? versions.params.previousVersions.concat(versions.params.currentVersion)
+	: [versions.params.currentVersion];
+
+const downloadConfigs = [];
+
+docVersions.forEach((version) => {
+	version = version === versions.params.currentVersion ? "" : "-" + version;
+	const downloadConfig = {
+		url: `https://fluidframework.blob.core.windows.net/api-extractor-json/latest${version}.tar.gz`,
+		destination: `../_api-extractor-temp${version}/doc-models/`,
+	};
+	downloadConfigs.push(downloadConfig);
+	console.log("DOWNLOAD CONFIGS", downloadConfigs);
+});
+
+(async () => {
+	await Promise.all(
+		downloadConfigs.map((config) => {
+			download(config.url, config.destination, { extract: true });
+		}),
+	);
+})();
```

Comment:
I don't think you need this outer async callback. This can probably be:
```suggestion
await Promise.all(
	downloadConfigs.map((config) => {
		download(config.url, config.destination, { extract: true });
}).then(
	() =>{
		console.log(...);
		process.exit(0);
	},
	(error) => {
		console.error(...);
		process.exit(1);
	},
);
```